decidim-meetings/app/types/decidim/meetings/agenda_item_type.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Meetings
```

Comment:
Again, there are many PRs with the same decidim-meetings, accountability, decidim-forms, proposals additions. Why have so many equal PRs? where's the difference between all of them?

Reply:
Because we built this on top of meetings. It should be clean now